drivers/ioexpander/gpio.c

```diff
@@ -578,47 +568,16 @@ int gpio_pin_register(FAR struct gpio_dev_s *dev, int minor)
  * Name: gpio_pin_unregister
  *
  * Description:
- *   Unregister GPIO pin device driver.
- *
- *   - Input pin types will be registered at /dev/gpinN
- *   - Output pin types will be registered at /dev/gpoutN
- *   - Interrupt pin types will be registered at /dev/gpintN
- *
- *   Where N is the provided minor number in the range of 0-99.
- *
+ *   Unregister GPIO pin device driver at /dev/gpioN, where N is the provided
+ *   minor number in the range of 0-99.
  *
  ****************************************************************************/
 
 void gpio_pin_unregister(FAR struct gpio_dev_s *dev, int minor)
 {
-  FAR const char *fmt;
   char devname[16];
```

Comment:
```suggestion
  char devname[32];
```
If we consider the max value for int minor, it requires more than 16 characters.